src/main/java/org/eclipse/foundation/marketplace/dto/Category.java

```diff
@@ -69,57 +65,49 @@ public void setUrl(String url) {
 		this.url = url;
 	}
 
-	/**
-	 * @return the marketIds
-	 */
-	public List<Integer> getMarketIds() {
-		return new ArrayList<>(marketIds);
-	}
-
-	/**
-	 * @param marketIds the marketIds to set
-	 */
-	public void setMarketIds(List<Integer> marketIds) {
-		this.marketIds = new ArrayList<>(marketIds);
-	}
-
 	@Override
 	public int hashCode() {
 		final int prime = 31;
 		int result = 1;
-		result = prime * result + id;
-		result = prime * result + ((marketIds == null) ? 0 : marketIds.hashCode());
+		result = prime * result + ((id == null) ? 0 : id.hashCode());
 		result = prime * result + ((name == null) ? 0 : name.hashCode());
 		result = prime * result + ((url == null) ? 0 : url.hashCode());
 		return result;
 	}
 
 	@Override
 	public boolean equals(Object obj) {
-		if (this == obj)
+		// check basic object properties
```

Comment:
Don't code equals by yourself, use Objects.equals https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#equals-java.lang.Object-java.lang.Object-

Reply:
I used the auto-generated equals from Eclipse. I think there was an option to use Object.equals instead. I'll try that instead.